docs/university/foundations/webstudio-dashboard.md

```diff
@@ -0,0 +1,17 @@
+# Webstudio Dashboard
```

Comment:
I still feel conflicted about this file a bit, I think it should be just answering how to do certain things that may be not as trivial as, the goal shouldn't be to simply describe everything user can see on the screen, because this is hard to maintain, things will change over time.